app/inline-scripts/load-ads.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+	var getterAdsQueue = [];
+	var adsLoaded = false;
+
+	function onAdsLoaded() {
+		adsLoaded = true;
+
+		getterAdsQueue.forEach(function (queuedCallback) {
+			queuedCallback();
+		});
+
+		getterAdsQueue = [];
+	}
+
+	window.getInstantGlobal('wgSitewideDisableAdsOnMercury', function (wgSitewideDisableAdsOnMercury) {
+		if (wgSitewideDisableAdsOnMercury || (new URL(document.location)).searchParams.get('noexternals')) {
```

Comment:
`noexternals` is handled in https://github.com/Wikia/mobile-wiki/pull/489/files#diff-2168f026d657db1526ae92392f844a3fR91. Isn't it enough?

Reply:
It's required to avoid loading ads code when `noexternals` is present.